platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -132,7 +132,60 @@ include::sample[dir="snippets/bestPractices/usePluginsBlock-do/groovy",files="bu
 <<plugins.adoc#sec:using_plugins,Using Plugins>>
 
 === Tags
+`#structuring-builds`
 
+[[use_convention_plugins]]
+== Use Convention Plugins for Common Build Logic
+
+Use convention plugins to encapsulate and reuse shared build logic across multiple projects in your build.
+
+=== Explanation
+
+Instead of duplicating configuration across multiple `build.gradle.kts` files, you can move common logic into a reusable plugin. This approach offers several benefits:
```

Comment:
`build.gradle.kts` 

Maybe I should use "build script" instead?

Reply:
Yes, better to not seem to limit this to Kotlin builds.